logstash.dockerfile

```diff
@@ -1,4 +1,6 @@
-FROM docker.elastic.co/logstash/logstash:7.6.1
+# Before building this dockerfile, remember to build the logstash-filter-usage_type plugin: `cd logstash-plugins/usagetype-plugin && ./gradlew gem && mv *.gem ../../ && mv *.gemspec ../../`
+# For more on building logstash plugins, see: https://www.elastic.co/guide/en/logstash/7.x/java-filter-plugin.html#_running_the_gradle_packaging_task_3
+FROM docker.elastic.co/logstash/logstash:7.7.0
```

Comment:
Is there a way in a dockerfile to include the build requirements mentioned in the comments?  

If not, maybe including a small wrapper shell script that builds the requirements and then builds the dockerfile?

Reply:
The  Usage Type plugin is meant to be built outside the container, for ease of development and testing. A wrapper script could be added at a future date but will not be included in this PR.

Reply:
Ah okay - is a note about this requirement in or needed in the README?  I'm just thinking if someone tries to run this dockerfile before the Usage Type plugin requirement and it doesn't work but they don't think to look in the dockerfile itself for this documentation.

Reply:
I can add a note to the README.

Reply:
README is updated.